### ISSUE TITLE
openwrt provider no longer shows the content of the files

### ISSUE THREAD

**Issue by mikegleasonjr** [ytti/oxidized#2968]
All the sections like:

```
##### File: /some/file #####
```
Are empty and aren't actually displaying the file content...

Here's what my config now outputs (see towards the end):

```
##### Info: /etc/banner #####
#  _______                     ________        __
# |       |.-----.-----.-----.|  |  |  |.----.|  |_
# |   -   ||  _  |  -__|     ||  |  |  ||   _||   _|
# |_______||   __|_____|__|__||________||__|  |____|
#          |__| W I R E L E S S   F R E E D O M
# -----------------------------------------------------
# OpenWrt 23.05.0, r23497-6637af95aa
# -----------------------------------------------------
#
#AO Build@2023.10.17 (manual build)
#
##### Info: /proc/cpuinfo #####
#processor	: 0
#BogoMIPS	: 48.00
#Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 cpuid
#CPU implementer	: 0x41
#CPU architecture: 8
#CPU variant	: 0x0
#CPU part	: 0xd03
#CPU revision	: 4
#
#processor	: 1
#BogoMIPS	: 48.00
#Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 cpuid
#CPU implementer	: 0x41
#CPU architecture: 8
#CPU variant	: 0x0
#CPU part	: 0xd03
#CPU revision	: 4
#
#processor	: 2
#BogoMIPS	: 48.00
#Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 cpuid
#CPU implementer	: 0x41
#CPU architecture: 8
#CPU variant	: 0x0
#CPU part	: 0xd03
#CPU revision	: 4
#
#processor	: 3
#BogoMIPS	: 48.00
#Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 cpuid
#CPU implementer	: 0x41
#CPU architecture: 8
#CPU variant	: 0x0
#CPU part	: 0xd03
#CPU revision	: 4
#
#processor	: 4
#BogoMIPS	: 48.00
#Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 cpuid
#CPU implementer	: 0x41
#CPU architecture: 8
#CPU variant	: 0x0
#CPU part	: 0xd08
#CPU revision	: 2
#
#processor	: 5
#BogoMIPS	: 48.00
#Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32 cpuid
#CPU implementer	: 0x41
#CPU architecture: 8
#CPU variant	: 0x0
#CPU part	: 0xd08
#CPU revision	: 2
#
##### Info: /etc/openwrt_release #####
#DISTRIB_ID='OpenWrt'
#DISTRIB_RELEASE='23.05.0'
#DISTRIB_REVISION='r23497-6637af95aa'
#DISTRIB_TARGET='rockchip/armv8'
#DISTRIB_ARCH='aarch64_generic'
#DISTRIB_TAINTS=''
#DISTRIB_DESCRIPTION='AO Build@2023.10.17 (manual build)'
##### Info: sysupgrade -l #####
#/etc/config/ddns
#/etc/config/dhcp
#/etc/config/dropbear
#/etc/config/firewall
#/etc/config/irqbalance
#/etc/config/luci
#/etc/config/network
#/etc/config/ntpclient
#/etc/config/openssl
#/etc/config/rpcd
#/etc/config/sqm
#/etc/config/system
#/etc/config/ucitrack
#/etc/config/uhttpd
#/etc/crontabs/root
#/etc/dropbear/authorized_keys
#/etc/dropbear/dropbear_ed25519_host_key
#/etc/dropbear/dropbear_rsa_host_key
#/etc/group
#/etc/hosts
#/etc/inittab
#/etc/luci-uploads/.placeholder
#/etc/nftables.d/10-custom-filter-chains.nft
#/etc/nftables.d/README
#/etc/opkg/keys/b5043e70f9a75cde
#/etc/passwd
#/etc/profile
#/etc/rc.local
#/etc/shadow
#/etc/shells
#/etc/shinit
#/etc/sysctl.conf
#/etc/uhttpd.crt
#/etc/uhttpd.key
#/etc/uhttpd.key
#/etc/uhttpd.crt
##### Info: /proc/mtd #####
#dev:    size   erasesize  name
##### File: /etc/config/ddns #####

##### File: /etc/config/dhcp #####

##### File: /etc/config/dropbear #####

##### File: /etc/config/firewall #####

##### File: /etc/config/irqbalance #####

##### File: /etc/config/luci #####

##### File: /etc/config/network #####

##### File: /etc/config/ntpclient #####

##### File: /etc/config/openssl #####

##### File: /etc/config/rpcd #####

##### File: /etc/config/sqm #####

##### File: /etc/config/system #####

##### File: /etc/config/ucitrack #####

##### File: /etc/config/uhttpd #####

##### File: /etc/crontabs/root #####

##### File: /etc/dropbear/authorized_keys #####

##### File: /etc/dropbear/dropbear_ed25519_host_key #####

##### Binary file: /etc/dropbear/dropbear_rsa_host_key #####
#Decode using 'echo -en <data> | gzip -dc > /etc/dropbear/dropbear_rsa_host_key'

##### File: /etc/group #####

##### File: /etc/hosts #####

##### File: /etc/inittab #####

##### File: /etc/luci-uploads/.placeholder #####

##### File: /etc/nftables.d/10-custom-filter-chains.nft #####

##### File: /etc/nftables.d/README #####

##### File: /etc/opkg/keys/b5043e70f9a75cde #####

##### File: /etc/passwd #####

##### File: /etc/profile #####

##### File: /etc/rc.local #####

##### File: /etc/shadow #####

##### File: /etc/shells #####

##### File: /etc/shinit #####

##### File: /etc/sysctl.conf #####

##### File: /etc/uhttpd.crt #####

##### File: /etc/uhttpd.key #####

##### File: /etc/uhttpd.key #####

##### File: /etc/uhttpd.crt #####
```

Here's my ssh prompt on the openwrt box:

```
BusyBox v1.36.1 (2023-10-09 21:45:35 UTC) built-in shell (ash)

  _______                     ________        __
 |       |.-----.-----.-----.|  |  |  |.----.|  |_
 |   -   ||  _  |  -__|     ||  |  |  ||   _||   _|
 |_______||   __|_____|__|__||________||__|  |____|
          |__| W I R E L E S S   F R E E D O M
 -----------------------------------------------------
 OpenWrt 23.05.0, r23497-6637af95aa
 -----------------------------------------------------

AO Build@2023.10.17 (manual build)

root@openwrt:~# 
```

Here are the logs:

```
D, [2023-10-18T12:38:24.274120 #29] DEBUG -- : lib/oxidized/input/ssh.rb: Connecting to <redacted>
D, [2023-10-18T12:38:24.274207 #29] DEBUG -- : AUTH METHODS::["none", "publickey", "password"]
D, [2023-10-18T12:38:24.475067 #29] DEBUG -- : lib/oxidized/input/cli.rb: Running post_login commands at <redacted>
D, [2023-10-18T12:38:24.475136 #29] DEBUG -- : lib/oxidized/model/model.rb Collecting commands' outputs
D, [2023-10-18T12:38:24.475171 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/banner
D, [2023-10-18T12:38:24.475198 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/banner @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:24.484642 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /proc/cpuinfo
D, [2023-10-18T12:38:24.484687 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /proc/cpuinfo @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:24.546410 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/openwrt_release
D, [2023-10-18T12:38:24.546478 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/openwrt_release @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:24.605831 #29] DEBUG -- : lib/oxidized/model/model.rb Executing sysupgrade -l
D, [2023-10-18T12:38:24.605892 #29] DEBUG -- : lib/oxidized/input/ssh.rb sysupgrade -l @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:24.868344 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /proc/mtd
D, [2023-10-18T12:38:24.868490 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /proc/mtd @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:24.925976 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export ddns
D, [2023-10-18T12:38:24.926034 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export ddns @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:24.987574 #29] DEBUG -- : Exporting uci config - ddns
D, [2023-10-18T12:38:24.987721 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export dhcp
D, [2023-10-18T12:38:24.987769 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export dhcp @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.047711 #29] DEBUG -- : Exporting uci config - dhcp
D, [2023-10-18T12:38:25.047783 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export dropbear
D, [2023-10-18T12:38:25.047803 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export dropbear @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.106376 #29] DEBUG -- : Exporting uci config - dropbear
D, [2023-10-18T12:38:25.106439 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export firewall
D, [2023-10-18T12:38:25.106456 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export firewall @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.167329 #29] DEBUG -- : Exporting uci config - firewall
D, [2023-10-18T12:38:25.167396 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export irqbalance
D, [2023-10-18T12:38:25.167413 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export irqbalance @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.225985 #29] DEBUG -- : Exporting uci config - irqbalance
D, [2023-10-18T12:38:25.226077 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export luci
D, [2023-10-18T12:38:25.226106 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export luci @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.275118 #29] DEBUG -- : lib/oxidized/worker.rb: 1 jobs running in parallel
D, [2023-10-18T12:38:25.288040 #29] DEBUG -- : Exporting uci config - luci
D, [2023-10-18T12:38:25.288179 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export network
D, [2023-10-18T12:38:25.288227 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export network @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.347400 #29] DEBUG -- : Exporting uci config - network
D, [2023-10-18T12:38:25.347489 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export ntpclient
D, [2023-10-18T12:38:25.347517 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export ntpclient @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.407421 #29] DEBUG -- : Exporting uci config - ntpclient
D, [2023-10-18T12:38:25.407572 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export openssl
D, [2023-10-18T12:38:25.407621 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export openssl @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.467587 #29] DEBUG -- : Exporting uci config - openssl
D, [2023-10-18T12:38:25.467733 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export rpcd
D, [2023-10-18T12:38:25.467782 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export rpcd @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.527456 #29] DEBUG -- : Exporting uci config - rpcd
D, [2023-10-18T12:38:25.527613 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export sqm
D, [2023-10-18T12:38:25.527665 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export sqm @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.587426 #29] DEBUG -- : Exporting uci config - sqm
D, [2023-10-18T12:38:25.587585 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export system
D, [2023-10-18T12:38:25.587650 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export system @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.647549 #29] DEBUG -- : Exporting uci config - system
D, [2023-10-18T12:38:25.647676 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export ucitrack
D, [2023-10-18T12:38:25.647725 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export ucitrack @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.707898 #29] DEBUG -- : Exporting uci config - ucitrack
D, [2023-10-18T12:38:25.708042 #29] DEBUG -- : lib/oxidized/model/model.rb Executing uci export uhttpd
D, [2023-10-18T12:38:25.708091 #29] DEBUG -- : lib/oxidized/input/ssh.rb uci export uhttpd @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.767259 #29] DEBUG -- : Exporting uci config - uhttpd
D, [2023-10-18T12:38:25.767443 #29] DEBUG -- : Exporting file - /etc/crontabs/root
D, [2023-10-18T12:38:25.767505 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/crontabs/root
D, [2023-10-18T12:38:25.767551 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/crontabs/root @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.827070 #29] DEBUG -- : Exporting file - /etc/dropbear/authorized_keys
D, [2023-10-18T12:38:25.827189 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/dropbear/authorized_keys
D, [2023-10-18T12:38:25.827300 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/dropbear/authorized_keys @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.887311 #29] DEBUG -- : Exporting file - /etc/dropbear/dropbear_ed25519_host_key
D, [2023-10-18T12:38:25.887421 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/dropbear/dropbear_ed25519_host_key
D, [2023-10-18T12:38:25.887470 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/dropbear/dropbear_ed25519_host_key @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:25.946367 #29] DEBUG -- : Exporting binary file - /etc/dropbear/dropbear_rsa_host_key
D, [2023-10-18T12:38:25.946460 #29] DEBUG -- : lib/oxidized/model/model.rb Executing gzip -c /etc/dropbear/dropbear_rsa_host_key | hexdump -ve '1/1 "_x%.2x"' | tr _ \
D, [2023-10-18T12:38:25.946505 #29] DEBUG -- : lib/oxidized/input/ssh.rb gzip -c /etc/dropbear/dropbear_rsa_host_key | hexdump -ve '1/1 "_x%.2x"' | tr _ \ @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.011585 #29] DEBUG -- : Exporting file - /etc/group
D, [2023-10-18T12:38:26.011723 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/group
D, [2023-10-18T12:38:26.011769 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/group @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.066316 #29] DEBUG -- : Exporting file - /etc/hosts
D, [2023-10-18T12:38:26.066380 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/hosts
D, [2023-10-18T12:38:26.066403 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/hosts @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.125286 #29] DEBUG -- : Exporting file - /etc/inittab
D, [2023-10-18T12:38:26.125366 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/inittab
D, [2023-10-18T12:38:26.125387 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/inittab @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.186675 #29] DEBUG -- : Exporting file - /etc/luci-uploads/.placeholder
D, [2023-10-18T12:38:26.186807 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/luci-uploads/.placeholder
D, [2023-10-18T12:38:26.186828 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/luci-uploads/.placeholder @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.246471 #29] DEBUG -- : Exporting file - /etc/nftables.d/10-custom-filter-chains.nft
D, [2023-10-18T12:38:26.246538 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/nftables.d/10-custom-filter-chains.nft
D, [2023-10-18T12:38:26.246562 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/nftables.d/10-custom-filter-chains.nft @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.276359 #29] DEBUG -- : lib/oxidized/worker.rb: 1 jobs running in parallel
D, [2023-10-18T12:38:26.306194 #29] DEBUG -- : Exporting file - /etc/nftables.d/README
D, [2023-10-18T12:38:26.306289 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/nftables.d/README
D, [2023-10-18T12:38:26.306310 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/nftables.d/README @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.365727 #29] DEBUG -- : Exporting file - /etc/opkg/keys/b5043e70f9a75cde
D, [2023-10-18T12:38:26.365805 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/opkg/keys/b5043e70f9a75cde
D, [2023-10-18T12:38:26.365825 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/opkg/keys/b5043e70f9a75cde @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.425552 #29] DEBUG -- : Exporting file - /etc/passwd
D, [2023-10-18T12:38:26.425593 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/passwd
D, [2023-10-18T12:38:26.425614 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/passwd @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.485655 #29] DEBUG -- : Exporting file - /etc/profile
D, [2023-10-18T12:38:26.485728 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/profile
D, [2023-10-18T12:38:26.485751 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/profile @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.545674 #29] DEBUG -- : Exporting file - /etc/rc.local
D, [2023-10-18T12:38:26.545714 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/rc.local
D, [2023-10-18T12:38:26.545735 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/rc.local @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.606505 #29] DEBUG -- : Exporting file - /etc/shadow
D, [2023-10-18T12:38:26.606591 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/shadow
D, [2023-10-18T12:38:26.606618 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/shadow @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.665731 #29] DEBUG -- : Exporting file - /etc/shells
D, [2023-10-18T12:38:26.665772 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/shells
D, [2023-10-18T12:38:26.665792 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/shells @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.726494 #29] DEBUG -- : Exporting file - /etc/shinit
D, [2023-10-18T12:38:26.726610 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/shinit
D, [2023-10-18T12:38:26.726627 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/shinit @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.786262 #29] DEBUG -- : Exporting file - /etc/sysctl.conf
D, [2023-10-18T12:38:26.786357 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/sysctl.conf
D, [2023-10-18T12:38:26.786384 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/sysctl.conf @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.846188 #29] DEBUG -- : Exporting file - /etc/uhttpd.crt
D, [2023-10-18T12:38:26.846262 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/uhttpd.crt
D, [2023-10-18T12:38:26.846281 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/uhttpd.crt @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.906598 #29] DEBUG -- : Exporting file - /etc/uhttpd.key
D, [2023-10-18T12:38:26.906709 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/uhttpd.key
D, [2023-10-18T12:38:26.906737 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/uhttpd.key @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:26.966933 #29] DEBUG -- : Exporting file - /etc/uhttpd.key
D, [2023-10-18T12:38:26.967082 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/uhttpd.key
D, [2023-10-18T12:38:26.967113 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/uhttpd.key @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:27.025665 #29] DEBUG -- : Exporting file - /etc/uhttpd.crt
D, [2023-10-18T12:38:27.025738 #29] DEBUG -- : lib/oxidized/model/model.rb Executing cat /etc/uhttpd.crt
D, [2023-10-18T12:38:27.025761 #29] DEBUG -- : lib/oxidized/input/ssh.rb cat /etc/uhttpd.crt @ <redacted> with expect: /^[^#]+#/
D, [2023-10-18T12:38:27.086420 #29] DEBUG -- : lib/oxidized/input/cli.rb Running pre_logout commands at <redacted>
D, [2023-10-18T12:38:27.087026 #29] DEBUG -- : lib/oxidized/node.rb: Oxidized::SSH ran for <redacted> successfully
D, [2023-10-18T12:38:27.087069 #29] DEBUG -- : lib/oxidized/job.rb: Config fetched for <redacted> at 2023-10-18 12:38:27 UTC
I, [2023-10-18T12:38:27.280437 #29]  INFO -- : Configuration updated for /<redacted>
```

**Comment by mikegleasonjr** [comment]
Also all the `uci export <something>` don't show up in the result generated..

**Comment by mikegleasonjr** [comment]
Seems like related to https://github.com/ytti/oxidized/issues/2957

**Comment by kbon** [comment]
Just want to add a +1 here, I'm facing the same, fresh setup of oxidized using latest "official" ytti Docker container `oxidized/oxidized:0.29.1-58-g8677c61`

**Comment by robertcheramy** [comment]
This issue will be fixed with PR #2982
